modules/core/server/services/influx.server.service.js

```diff
@@ -44,12 +46,12 @@ exports.getClient = function(callback) {
  * influxdb (camelCase)
  * @param {Object} fields - key: value pairs will be saved in influxdb as field
  * key: field value
- * @param {number|Date} [fields.time=new Date()] - time of measurement with precision ms
+ * @param {number|Date} [fields.time=new Date()] - time of measurement with precision nanosecond
```

Comment:
@simison do you prefer me committing the changes directly, or leaving it for you to decide and execute?

This is a pretty general question, really. The first option opens a space for both (a mess and misunderstanding) and (a higher and exciting level of collaboration).